nats/src/jetstream/types.rs

```diff
@@ -708,11 +710,23 @@ pub struct ConsumerInfo {
     pub push_bound: bool,
 }
 
-/// Information about the consumer's associated `JetStream` cluster
+/// Information about the stream's, consumer's associated `JetStream` cluster
 #[derive(Debug, Default, Serialize, Deserialize, Clone, PartialEq, Eq)]
 pub struct ClusterInfo {
+    /// The name of cluster
+    pub name: String,
     /// The leader of the cluster
     pub leader: String,
+    /// The replica state of cluster
+    pub replicas: Vec<ClusterReplicas>
+}
+
+#[derive(Debug, Default, Serialize, Deserialize, Clone, PartialEq, Eq)]
+pub struct ClusterReplicas {
```

Comment:
Naming suggest plural, maybe keep schema's naming.

```suggestion
pub struct PeerInfo {
```

Reply:
Rename and Succeed for add explain Mapping !